clickplc/driver.py

```diff
@@ -715,6 +941,25 @@ async def _set_td(self, start: int, data: list[int]):
 
         await self.write_registers(address, values)
 
+    async def _set_ctd(self, start: int, data: list[int]):
```

Comment:
Interesting, I had thought it was a bad idea to set CTD registers but apparently it's OK.

This should have a new test for the roundtrip.